packages/react-ui/src/Titles/index.js

```diff
@@ -38,10 +44,21 @@ const StyledPageTitle = styled.h1`
   font-size: ${fonts.sizes.headings.large};
   font-family: "Merriweather", serif;
   line-height: ${fonts.lineHeightTitle};
-  ${({ leftStripped }) => {
+  ${({ leftStripped, shift }) => {
     if (leftStripped) {
+      if (shift) {
+        return css`
+          padding-left: ${shift};
+          @media (max-width: ${breakpoints.mobile}) {
+            padding-left: ${spacings.base};
+          }
+        `;
+      }
       return css`
         padding-left: ${spacings.large};
+        @media (max-width: ${breakpoints.mobile}) {
```

Comment:
Better to add a ternary on padding-left
```
padding-left: ${shift?shift:spacings.large};
```
To avoid duplicate

Reply:
bien vu !